### ISSUE TITLE
How to add categories for Special Projects/Artists?

### ISSUE THREAD

**Issue by janislejins** [jsonresume/resume-schema#247]
Hey My name is Janis Lejins and I'm an artist (which most people think means unemployed)
But contrary to popular belief Artists use cvs and resumes all the time - In fact because we have to apply to third parties (galleries, granting bodies, patrons, etc) more often we actually have to generate Resumes more often. 

Every-time you enter for an exhibition, grant, residency, award etc you are usually required to provide an artists cv.

I know personally being able to use something like http://registry.jsonresume.org/ would make artists lives a lot easier.

Obviously as well as having 'Work Experience' field we have exhibition history (aka in jobs we have done as artists) we more frequently do one of things generally split between 'Solo Exhibitions' , 'Group Exhibitions' and 'Other Projects':

It might be useful to have a projects style field:

```
    "Projects": {
      "type": "array",
      "additionalItems": false,
      "items": {
        "type": "object",
        "additionalProperties": true,
        "properties": {
          "Gallery": {
            "type": "string",
            "description": "Museum of Modern Art"
          },
          "Exhibition Title": {
            "type": "string",
            "description": "Australian Artists in the 21st Century"
          },
          "Exhibition type": {
            "type": "string",
            "description": "Specify Group, Solo or Special Commission"
          },
          "website": {
            "type": "string",
            "description": "e.g. http://moma.com",
            "format": "uri"
          },
          "startDate": {
            "type": "string",
            "description": "resume.json uses the ISO 8601 date standard e.g. 2014-06-29",
            "format": "date"
          },
          "endDate": {
            "type": "string",
            "description": "e.g. 2012-06-29",
            "format": "date"
          },
          "summary": {
            "type": "string",
            "description": "Give an overview of your responsibilities at the company"
          },
          "highlights": {
            "type": "array",
            "description": "Specify multiple accomplishments/reviews/media coverage",
            "additionalItems": false,
            "items": {
              "type": "string",
              "description": "e.g. Visionary - New York Times"
            }
          }
        }
      }
    },
```

**Comment by stp-ip** [comment]
The proposed idea for the next version is to have a general projects section, with subprojects. Each project could reflect  'Solo Exhibitions', 'Group Exhibitions' and 'Other Projects'. Subprojects then could specify the specific exhibitions and/or items. Does that make sense?

More infos are available in this issue: #226

The new scheme is coming along a bit slower than anticipated, but there is still work going on. Always great to have new insights into the needs of the users.

**Comment by wizzor** [comment]
Hi,
I support this strongly. As an employee in a consultancy house, project references are important. We could start managing our employees CVs in this format, but only if we can list projects.

**Comment by nelg** [comment]
good idea.. need it too

**Comment by stp-ip** [comment]
First iteration of projects is now merged into the v1.0.0 branch via https://github.com/jsonresume/resume-schema/pull/254.

I'm closing this issue for now, but if there is anything you would like to be added to this first iteration please open up a new issue to improve the latest implementation. Also PRs are welcome, but might need some discussion to be included.
